pyomo/environ/__init__.py

```diff
@@ -38,12 +38,13 @@ def _do_import(pkg_name):
 #
 #
 # These packages also contain plugins that need to be loaded, but
-# we silently ignore any import errors because these 
+# we silently ignore any import errors because these
 # packages are optional and/or under development.
 #
 _optional_packages = set([
     'pyomo.contrib.example',
-    'pyomo.contrib.preprocessing'])
+    'pyomo.contrib.preprocessing',
```

Comment:
What are these _optional_packages used for?

Reply:
The `_optional_packages` import the necessary contrib modules, such as the transformation plugins that I have under `pyomo.contrib.preprocessing`. The comment lines above describe its functionality.

Reply:
OK, the comment is confusing since it says they "need to be loaded", but also says that they "are optional". No biggie. I am fine with it.